src/Modules/Shared/PowerShellGet/PSModule.psm1

```diff
@@ -122,7 +122,7 @@ $script:NuGetProviderName = "NuGet"
 $script:NuGetProviderVersion  = [Version]'2.8.5.201'
 
 $script:SupportsPSModulesFeatureName="supports-powershell-modules"
-$script:FastPackRefHastable = @{}
+$script:FastPackRefHashtable = @{}

```

Comment:
Can we separate script files (.psm1) into a separate PR?
C# code has static compile checks, but psm1 files would need more attention.